packages/terra-consumer-nav/src/components/user-profile/UserProfile.jsx

```diff
@@ -46,37 +56,53 @@ const propTypes = {
 };
 
 const defaultProps = {
-  profile: {
-    avatar: null,
-    profileLinks: [],
-  },
+  avatar: <IconPerson />,
+  profileLinks: [],
+  isSignIn: false,
+  id: '--terra-conumser--nav-profile-button',
 };
 
-
 const UserProfile = ({
-  name, avatar, id, signoutUrl, profileLinks, handleClick, intl, ...customProps
+  userName, avatar, id, signoutUrl, signinUrl, isSignIn, profileLinks, handleClick, intl, ...customProps
 }) => {
-  const content = (
-    <div>
-      <ProfileLinks linkItems={profileLinks} />
-      <Button className={cx('link', 'signout-border')} href={signoutUrl}>
-        <FormattedMessage id="nav_signout" />
+  let ret;
```

Comment:
whats a ret?

Reply:
a shortened form of return when i'm typing fast and not thinking of good var names haha